packages/usa-embed-container/src/styles/_usa-embed-container.scss

```diff
@@ -1,24 +1,5 @@
 @use "uswds-core" as *;
 
-$aspect-height: 9;
-$aspect-width: 16;
-
-.usa-embed-container {
-  iframe,
-  object,
-  embed {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-  }
-}
-
 .usa-embed-container {
-  @include embed-container($aspect-height, $aspect-width);
-  position: relative;
-  height: 0;
-  overflow: hidden;
-  max-width: 100%;
+  @include add-aspect("16x9");
```

Comment:
Correct. The `.usa-embed-container` and `.add-aspect-` classes are now equivalent, and `@mixin embed-container` is no longer needed or referenced in our code base. I kept the mixin available for backwards compatibility in case users call that mixin in their project code. Is it safe to remove?

Reply:
Good consideration! In the past we've moved deprecated features to `_deprecated.scss`.